packages/terra-paginator/README.md

```diff
@@ -0,0 +1,25 @@
+# Terra Paginator
+
+
+[![NPM version](http://img.shields.io/npm/v/terra-paginator.svg)](https://www.npmjs.org/package/terra-paginator)
+[![Build Status](https://travis-ci.org/cerner/terra-core.svg?branch=master)](https://travis-ci.org/cerner/terra-core)
+
+Paginator to be used for data sets of known and unknown size. Provides first, last, previous, next, and paged functionality.
+
+- [Getting Started](#getting-started)
+- [Documentation](https://github.com/cerner/terra-core/tree/master/packages/terra-paginator/docs)
+- [LICENSE](#license)
+
+## Getting Started
+
+- Install from [npmjs](https://www.npmjs.com): `npm install terra-paginator`
+
+## LICENSE
+
+Copyright 2017 Cerner Innovation, Inc.
```

Comment:
This should be 2018

Reply:
https://github.com/cerner/terra-core/pull/1445/commits/c619c108cd3f92c3084a93d5dc3d99e53bd717ba